packages/oauth/tsconfig.json

```diff
@@ -1,10 +1,11 @@
 {
   "compilerOptions": {
-    "declaration": true,
+    "declaration": false,
```

Comment:
What was the issue here?

Reply:
I think I changed that in troubleshooting (I don't remember what) and forgot to change it back.